docs/external_integrations/authorization_ory_kratos/handling_webhooks.md

```diff
@@ -0,0 +1,183 @@
+# Handling webhooks
+
+To effectively manage incoming webhooks from Ory Kratos, you can employ the [KratosWebHookHandlerBase] class. The following example demonstrates how to synchronize identity data received from Kratos through webhooks with your database using [MassTransit]. This assumes that your Kratos instance is properly configured to send webhooks to your application.
+
+## Packages
+
+| Package | Link | Application in section |
+| --- | ----------- | ----------- |
+| LeanCode.Kratos | [![NuGet version (LeanCode.Kratos)](https://img.shields.io/nuget/vpre/LeanCode.Kratos.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.Kratos/8.0.2260-preview/) | Webhook handling |
```

Comment:
Do we need to specifically link to `8.0.2260-preview` version? Maybe omitting this part of the path will just redirect the user to latest version?

Reply:
It will redirect user to latest version which is not in preview (or latest one if all versions are in preview), so it could be removed after corelib v8.0 is out of preview.